doc/en/components/grids/_shared/state-persistence.md

```diff
@@ -857,6 +1005,6 @@ state.setState(gridState.columnSelection);
 
 
 <!-- * [Pivot Grid Features](features.md) -->
-<!-- * [Pivot Grid Remote Operations](remote-operations.md) -->
+* [Pivot Grid Remote Operations](remote-operations.md)
```

Comment:
This seems to throw an error on build for react: warning InvalidFileLink: Invalid file link:(~/components/grids/pivot-grid/remote-operations.md).

Perhaps it should be removed for react since there's no such topic there.

Reply:
Yes, I was waiting so that react topic for pivot grid are done.

Reply:
I'm not sure we can have remote operations topic for react since it depends on noop strategies that are not exported for react/blazor.

Reply:
Okay. I will enable this only for WC then.